sqlx-core/src/transaction.rs

```diff
@@ -78,6 +85,26 @@ where
         })
     }
 
+    #[doc(hidden)]
+    pub fn begin_with<S>(
+        conn: impl Into<MaybePoolConnection<'c, DB>>,
+        sql: S,
```

Comment:
This can just be `Cow<'static, str>` here since it's not a public API. That will save on codgen.